tests/lazyread.hs

```diff
@@ -1,17 +0,0 @@
--- Ensure that readInt and readInteger over lazy ByteStrings are not
--- excessively strict.
-
-module Main (main) where
-
-import Data.ByteString.Char8 (pack)
-import Data.ByteString.Lazy.Char8 (readInt, readInteger)
-import Data.ByteString.Lazy.Internal (ByteString(..))
-
-main :: IO ()
```

Comment:
This also looks reasonable enough to keep around (and ideally include into the test suite).